packages/frontend/src/state/crab/hooks.ts

```diff
@@ -424,7 +424,11 @@ export const useCurrentCrabPositionValueV2 = () => {
     crabQueuedShares,
   ])
 
-  return { currentCrabPositionValue, currentCrabPositionValueInETH, isCrabPositionValueLoading }
+  return {
+    currentCrabPositionValue,
+    currentCrabPositionValueInETH,
+    isCrabPositionValueLoading: balLoading || isCrabPositionValueLoading,
```

Comment:
nice. I think we can remove [this](https://github.com/opynfinance/squeeth-monorepo/pull/886/files#diff-683cf1dadaa899aca98cf44d3316f0ecf254c9461253cf711576b771b532fd73R356) condition now

Reply:
Cant find the condition you are pointing at